Languages/es.xml

```diff
@@ -36,48 +36,48 @@ Gracias a: Oleada, Jhovany200, nomemolesteis@gmail.com por la traducción.
   </CCleanLangSecRefMap>
   <StringMaps>
     <String Target="文件(&amp;F)" Link="&amp;Archivo"/>
-      <String Target="添加映像" Link="Añadir imagen"/>
-        <String Target="释放映像&#9;Ctrl+N" Link="Aplicar imagen&#9;Ctrl+N"/>
-        <String Target="挂载映像&#9;Ctrl+Shift+N" Link="Montar imagen&#9;Ctrl+Mayús+N"/>
-        <String Target="添加路径" Link="Añadir ruta"/>
-      <String Target="卸载映像&#9;Delete" Link="Desmotar imagen&#9;Supr"/>
-      <String Target="保存映像&#9;Ctrl+S" Link="Guardar imagen&#9;Ctrl+S"/>
-      <String Target="另存为映像&#9;Ctrl+Shift+S" Link="Guardar imagen como&#9;Ctrl+Mayús+S"/>
-      <String Target="打开映像文件&#9;Ctrl+O" Link="Abrir archivo de imagen&#9;Ctrl+O"/>
-      <String Target="重新载入" Link="Recargar"/>
-      <String Target="格式转换" Link="Convertir formato"/>
-        <String Target="ESD解密" Link="Desencriptar ESD"/>
-      <String Target="取消任务&#9;Ctrl+Break" Link="Cancelar tarea&#9;Ctrl+Break"/>
+    <String Target="添加映像" Link="Añadir imagen"/>
```

Comment:
Don't remove ' '...

Reply:
do not remove it, it's just aligned.

Reply:
I need a clear structure, not a single-minded alignment. Thanks